pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -141,7 +142,9 @@ async def run(self):
             if pr:
                 message += f"\nA PR to update the assembly definition has been created/updated: {pr.html_url}"
             if assembly_type in [AssemblyTypes.PREVIEW, AssemblyTypes.CANDIDATE]:
-                message += f"\n This is a {assembly_type.name} release. Please run <https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/aos-cd-builds/job/build%252Fmicroshift_sync/|microshift_sync> to publish the build to mirror."
+                jenkins.start_microshift_sync(version=version, assembly=self.assembly)
```

Comment:
The `BUILD_VERSION` parameter expects `<major>.<minor>`, like `4.14`. However the `version` returned by `generate_microshift_version_release(release_name)` is the rpm version, which includes the `.z` part, like `4.14.1`.

Let's use value from `major, minor = util.isolate_major_minor_in_group(self.group)` instead.

Reply:
Fixed in https://github.com/openshift-eng/aos-cd-jobs/pull/3820/commits/fcbbe7322089d052618b9ccaf065e5f4537b738a